apps/issuer/managers.py

```diff
@@ -175,30 +174,42 @@ def create(self,
 
         # self.model would be a BadgeInstance
         new_instance = self.model(
-            public=False,
-            recipient_identifier=recipient_identifier,
-            badgeclass=badgeclass,
-            issuer=issuer,
-            **kwargs
+            public=False, recipient_identifier=recipient_identifier, badgeclass=badgeclass, issuer=issuer, **kwargs
         )
 
         with transaction.atomic():
             new_instance.save()
```

Comment:
Why do we need two saves?

Reply:
Before we can save the image to the new_instance we first need to save the instance to have it stored in the db, then we save it again to have the associated image linked.